src/internal/components/dropdown/__tests__/dropdown.test.tsx

```diff
@@ -88,6 +103,44 @@ describe('Dropdown Component', () => {
       expect(screen.getByTestId('after-dismiss')).toBeTruthy();
     });
   });
+
+  describe('dropdown focus events', () => {
+    test('fires focus and blur events when focus transitions from and to an outside element', async () => {
```

Comment:
This unit test was completely missing, so I added it in order to test both inside and outside cases.